src/services/qbit.js

```diff
@@ -227,10 +227,12 @@ class Qbit {
 
   getGlobalDownloadLimit() {
     return this.axios.get('/transfer/downloadLimit')
+      .then(res => res.data)
```

Comment:
I've seen you using await here and there and to uniform it this IMHO should be rewritten like:

```
 async getGlobalDownloadLimit() {
   const {data} = await this.axios.get('/transfer/downloadLimit')
   return data
```

Reply:
Sure, I don't really have that big of a preference.... I've used both in that class.